### DISCUSSION TITLE
[Feat Request]: Summary of Changes

### DISCUSSION THREAD

**Discussion by Noname1122** [RetroDriven/Pocket_Updater#12] [Implemented Ideas]
With so many cores in place, it's getting harder already to tell which cores or files have been added or changed in a given run of this tool. As such, if you're loading these files into a staging folder, instead of right onto the pocket, it's hard to know what all should be copied over. You have to read back through the verbose output of the tool to find out.

Would it be possible to add a summary (either of files added/changed or simply of Cores added/changed) to the bottom of the output as a summary? Or perhaps a toggle to have just that output vs the full verbose output?

Thanks!!

**Comment by RetroDriven** [comment]
This has been added to v1.3.3
